.github/workflows/ci.yml

```diff
@@ -62,8 +62,6 @@ jobs:
           DB_DATABASE: database/database.sqlite
         run: |
           php artisan db:seed
-      # - name: Execute tests (Unit and Feature tests) via PHPUnit
-      #   env:
-      #     DB_CONNECTION: sqlite
-      #     DB_DATABASE: database/database.sqlite
-      #   run: vendor/bin/phpunit
+      - name: Execute tests (Unit and Feature tests) via Pest
+        run: |
+          ./vendor/bin/pest
```

Comment:
Is calling `./vendor/bin/pest` or `php artisan test` the correct/recommended way to execute the tests?

Reply:
`./vendor/bin/pest` is here the "best" way. `php artisan test` is for development and debugging  (see https://laravel.com/docs/11.x/testing#running-tests)

Reply:
I tried both and didn't see a difference